src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,15 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine kamaz = new Track();
+        Machine cat = new Bulldozer();
```

Comment:
https://answear.ua/750427-caterpillar-vysokie-botinki-colorado.html?channable=e54191.NzUwNDI3&gclid=Cj0KCQjw1Iv0BRDaARIsAGTWD1sng1TxQCrTDQLBJEPQFQYDjtldaba9mM_dY6FKXavxAjqEEi2Dr1AaArKPEALw_wcB ))) you should give more specific name or  name them just as class